packages/core/admin/admin/src/content-manager/history/components/VersionsList.tsx

```diff
@@ -1,19 +1,226 @@
+import { Box, Flex, Typography } from '@strapi/design-system';
+import { useQueryParams } from '@strapi/helper-plugin';
+import { Contracts } from '@strapi/plugin-content-manager/_internal/shared';
+import { formatDistanceToNowStrict } from 'date-fns';
+import * as locales from 'date-fns/locale';
+import { stringify } from 'qs';
+import { type MessageDescriptor, useIntl } from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import { getDateFnsLocaleName } from '../../../utils/locales';
+import { getDisplayName } from '../../utils/users';
+
+/* -------------------------------------------------------------------------------------------------
+ * BlueChunks
+ * -----------------------------------------------------------------------------------------------*/
+
+const BlueChunks = (chunks: React.ReactNode) => (
+  <Typography textColor="primary600">{chunks}</Typography>
+);
+
+/* -------------------------------------------------------------------------------------------------
+ * VersionCard
+ * -----------------------------------------------------------------------------------------------*/
+
+type Color = React.ComponentProps<typeof Box>['color'];
+
+interface StatusData {
+  background: Color;
+  border: Color;
+  text: Color;
+  message: MessageDescriptor;
+}
+
+interface VersionCardProps {
+  version: Contracts.HistoryVersions.GetHistoryVersions.Response['data'][number];
+  isCurrent: boolean;
+}
+
+const VersionCard = ({ version, isCurrent }: VersionCardProps) => {
+  const { formatDate, formatMessage, locale } = useIntl();
+  const [{ query }] = useQueryParams<{ id?: string }>();
+
+  const isActive = query.id === version.id.toString();
+
+  const getStatusData = (): StatusData => {
+    switch (version.status) {
+      case 'draft':
+        return {
+          background: 'secondary100',
+          border: 'secondary200',
+          text: 'secondary700',
+          message: {
+            id: 'content-manager.containers.List.draft',
+            defaultMessage: 'Draft',
+          },
+        };
+      case 'modified':
+        return {
+          background: 'alternative100',
+          border: 'alternative200',
+          text: 'alternative700',
+          message: {
+            // TODO: check the translation key once D&P v5 is done
+            id: 'content-manager.containers.List.modified',
+            defaultMessage: 'Modified',
+          },
+        };
+      case 'published':
+      default:
+        return {
+          background: 'success100',
+          border: 'success200',
+          text: 'success700',
+          message: {
+            id: 'content-manager.containers.List.published',
+            defaultMessage: 'Published',
+          },
+        };
+    }
+  };
+
+  const statusData = getStatusData();
+
+  const distanceToNow = formatDistanceToNowStrict(new Date(version.createdAt), {
+    addSuffix: true,
+    locale: locales[getDateFnsLocaleName(locale)],
+  });
```

Comment:
We used a RelativeTime component from the helper-plugin for content-releases

https://github.com/strapi/strapi/blob/3d99eae71b67041695faf8533bf89f1558ad8b45/packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx#L401

Seems like a similar case here? Or does it not handle the locale and we should make an update over there?